xml/html_title.xml

```diff
@@ -3247,4 +3391,332 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:moinmo:moinmoin:-"/>
   </fingerprint>
 
-</fingerprints>
+  <fingerprint pattern="^(\w+) - BlackBox Management Console$">
+    <description>BlackBox Management Console</description>
+    <example host.name="ABC123">ABC123 - BlackBox Management Console</example>
+    <param pos="0" name="hw.vendor" value="BlackBox"/>
+    <param pos="0" name="hw.device" value="Device Server"/>
+    <param pos="0" name="hw.product" value="BlackBox"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Shelly Color Bulb$">
+    <description>Shelly Smart Bulb</description>
+    <example>Shelly Color Bulb</example>
+    <param pos="0" name="os.vendor" value="Cesanta"/>
+    <param pos="0" name="os.product" value="Mongoose"/>
+    <param pos="0" name="hw.vendor" value="Shelly"/>
+    <param pos="0" name="hw.device" value="Light Bulb"/>
+    <param pos="0" name="hw.product" value="Color Bulb"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Shelly (Plug|Dimmer(?: 2)?|Switch)$">
+    <description>Shelly Plug/Dimmer/Switch Devices</description>
+    <example hw.product="Plug">Shelly Plug</example>
+    <example hw.product="Dimmer">Shelly Dimmer</example>
+    <example hw.product="Dimmer 2">Shelly Dimmer 2</example>
+    <example hw.product="Switch">Shelly Switch</example>
+    <param pos="0" name="os.vendor" value="Cesanta"/>
+    <param pos="0" name="os.product" value="Mongoose"/>
+    <param pos="0" name="hw.vendor" value="Shelly"/>
+    <param pos="0" name="hw.device" value="Device"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Shelly[ ]?(RGBW2|1L)$">
+    <description>Shelly Lighting Controller</description>
+    <example hw.product="RGBW2">Shelly RGBW2</example>
+    <example hw.product="1L">Shelly1L</example>
+    <param pos="0" name="os.vendor" value="Cesanta"/>
+    <param pos="0" name="os.product" value="Mongoose"/>
+    <param pos="0" name="hw.vendor" value="Shelly"/>
+    <param pos="0" name="hw.device" value="Device"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+
+  <fingerprint pattern="^TinyGS Configuration$">
+    <description>TinyGS LoRa Ground Station Software</description>
+    <example>TinyGS Configuration</example>
+    <param pos="0" name="service.product" value="TinyGS"/>
+  </fingerprint>
+
+  <fingerprint pattern="^WLED$">
```

Comment:
Is this an actual vendor or just the generic wirelessly controlled LED devices?

Reply:
This is actually [open-source software](https://github.com/Aircoookie/WLED) running on an ESP embedded device for controlling certain brands/types of LEDs. Perhpas `Aircookie` would be more accurate for vendor...?